### ISSUE TITLE
Grenade

### ISSUE THREAD

**Issue by WongIong** [ZombieStriker/QualityArmory#161]
Grenade can shot an arrow.. XD when u are holding it on right hand.

**Comment by Lorenzo0111** [comment]
Can you send me a video of the bug?

**Comment by wongiong** [comment]
Sure. https://youtu.be/of8LydDyF90

**Comment by Lorenzo0111** [comment]
guns only works if they are on the right hand

**Comment by wongiong** [comment]
yes i know it. but what i mean is maybe this event should be canceled
